core/app/models/spree/stock_location.rb

```diff
@@ -12,6 +12,9 @@ class InvalidMovementError < StandardError; end
     belongs_to :state, class_name: 'Spree::State'
     belongs_to :country, class_name: 'Spree::Country'
 
+    has_many :shipping_method_stock_locations, class_name: Spree::ShippingMethodStockLocation
```

Comment:
I'm shutting down a stock location and destroying it in the admin. What should happen?